docs/organizations/managing-people.md

```diff
@@ -3,28 +3,30 @@
 Members of an organization can see the details of the repositories in that organization and add new repositories to be analyzed by Codacy. Besides this, Codacy only analyzes commits to **private repositories** from contributors who are members of the corresponding organization on Codacy.
 
 !!! important
-    Make sure that you invite or ask your team members to join your organization on Codacy so that Codacy analyzes their commits on private repositories.
+    -   Make sure that you invite or ask your team members to join your organization on Codacy so that Codacy analyzes their commits on private repositories.
 
-    Contributors who aren't part of your Git provider organization can't join your Codacy organization, but you should still add them to Codacy to analyze their commits on private repositories.
+    -   Contributors who aren't part of your Git provider organization can't join your organization on Codacy app, but you should still add them to your Codacy organization to analyze their commits on private repositories.
 
-To list and manage the members of your organization, open your organization **Settings**, page **People**. This page also shows when the organization members last logged in to Codacy:
+    -   In some Enterprise plans, Codacy automatically adds to the organization new people that commit on your private repositories and analyzes their commits.
```

Comment:
Are we setting on the 'Enterprise plans' idea? I guess that is fine, was just not expecting it

Reply:
Thank you so much for the input, @machadoit !

We don't want the product documentation to depend on specific billing plan names, such as Active Pro, because Sales can decide to change the billing rules at any time.

@AmhaBekele suggested using "Enterprise plans" in a generic way, which I agree with, as it's a term used in the software industry and we also use it in our [Pricing page](https://www.codacy.com/pricing): Enterprise plans are the indicated for large teams with Enterprise requirements, and are managed by Sales directly.

<img width="803" alt="image" src="https://user-images.githubusercontent.com/26439949/217570681-ae78deef-b4ea-405d-b244-9b0c5b59faa8.png">

Anyway, we can still discuss it with @AmhaBekele, if needed.